src/diff/index.js

```diff
@@ -158,15 +161,6 @@ export function diff(parentDom, newVNode, oldVNode, context, isSvg, excessDomChi
 		}
 
 		if (tmp = options.diffed) tmp(newVNode);
-
-		const hooks = newVNode._component && newVNode._component.__hooks;
-		if (hooks) {
-			hooks._handles.some(handle => {
-				if (handle.ref) handle.ref.current = handle.createHandle();
-			});
-			hooks._handles = [];
```

Comment:
Note: I removed this code by scheduling imperative handle creation as a layout effect, reusing the layout flush to call `createHandle` so you won't see it anywhere.